### DISCUSSION TITLE
모든 유저에게 UI의 Image가 뜨는 문제

### DISCUSSION THREAD

**Discussion by Park-sechang** [naverz/zepeto-studio-kor#1155] [5. Q&A]
특정한 오브젝트에 부딫히면 UI의 Image가 뜨는 방식인데, 

문제는 부딫히지 않은 다른 유저들에게도 뜹니다.

부딫힌 유저에게만 뜨게 할 수 있을까요?

![Screenshot_11](https://user-images.githubusercontent.com/99107908/183907519-e919274c-21d8-4ccc-b77e-238e860a842d.png)

**Comment by YSHFM** [comment]
레이어 설정을 해주시거나 태그로 관리하여 체크하시면 됩니다
레이어에 다른 충돌이 발생할 때 충돌관련 이벤트가 발생합니다.

**Comment by koldshine** [comment]
본인의 캐릭터가 충돌했을 때만 본인에게 보이면 될 것 같네요

trigger발동 함수에
if(collider.gameobject == ZepetoPlayers.instance.LocalPlayer.zepetoPlayer.character.gameobject){
    //이미지 보여줌
}
이런식으로요

**Reply by Park-sechang** [reply]
해결했습니다. 감사합니다!
